packages/p2p/src/pages/app.jsx

```diff
@@ -7,9 +7,9 @@ import { routes, WS } from '@deriv/shared';
 import ServerTime from 'Utils/server-time';
 import { waitWS } from 'Utils/websocket';
 import { useStores } from 'Stores';
-import AppContent from './app-content.jsx';
-import { setLanguage } from './i18next';
-import { ModalManager, ModalManagerContextProvider } from './modal-manager';
+import AppContent from 'Components/app-content.jsx';
+import { setLanguage } from 'Components/i18next';
+import { ModalManager, ModalManagerContextProvider } from 'Components/modal-manager/index.js';
```

Comment:
```suggestion
import { ModalManager, ModalManagerContextProvider } from 'Components/modal-manager';
```

Can it be like this?

Reply:
Haa yes thank you, the imports was auto-updated by vscode